macantine/etl/analysis.py

```diff
@@ -106,30 +106,38 @@ def flatten_central_kitchen_td(self):
         Split rows of central kitchen into a row for each satellite
         """
         self.df = self.df.set_index("id", drop=False)
+        new_rows = []
```

Comment:
```suggestion
        satellite_rows = []
```
pour être plus explicite

Reply:
done